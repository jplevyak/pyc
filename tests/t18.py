m = 1
class A:
  n = 2
  print m
  print n
print A.n
class B(A):
  o = 3
  print m
  print o
print B.n
A.n = 4
print A.n
print B.n
print A().n
print B().n
B.n = 5
print A.n
y = B();
print y.n
print y.o
