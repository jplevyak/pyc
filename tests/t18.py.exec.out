1
2
2
1
3
2
4
2
4
2
4
5
3
